<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>visualizarPedido</class>
 <widget class="QWidget" name="visualizarPedido">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualizar Pedido - [GeriFácil] </string>
  </property>
  <widget class="QPushButton" name="b_voltar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cliente:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabela">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>950</width>
     <height>500</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>114</number>
   </attribute>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>7</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>8</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>9</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>10</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>11</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>12</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>13</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Código</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Descrição</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Unidade</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Quantidade</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Custo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Venda</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Total</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Lucro</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="b_adicionar_produto">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>100</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Adicionar 
 Produto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_remover_produto">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>160</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 90, 90);</string>
   </property>
   <property name="text">
    <string>Remover 
 Produto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_imprimir">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>220</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(243, 247, 129);</string>
   </property>
   <property name="text">
    <string>Imprimir</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>590</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Margem de lucro:    R$ </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="in_margem_luco_rs">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>590</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="in_marge_lucro">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>590</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>590</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>630</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Pagamento:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_pagamento">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>630</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <item>
    <property name="text">
     <string>À VISTA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CHEQUE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7 DIAS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15 DIAS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30 DIAS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>35 DIAS</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="b_salvar">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>280</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>590</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Venda:  R$</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="in_total">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>590</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="in_nome_cliente">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Carga:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="in_nome_carga">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>350</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Após qualquer alteração feita no pedido 
do cliente,clicar no botão &quot;Salvar&quot; acima,
caso contrário,as modificações serão descartadas.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="in_total_custo">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>620</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>620</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Custo:  R$</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
